### PR TITLE
Privacy policy

### PR DESCRIPTION
**Pull request by niknetniko**

### CODE REVIEW

**Comment by niknetniko** [comment]
Merging this since I want to get the Assistant thing published soon-ish. Suggestions are always welcome if things are not clear or not correct.

File: server/static/website/privacy-policy.html [hunk 1]
```diff
@@ -1,141 +1,212 @@
...
+  <li>De <strong>Android-app</strong>, beschikbaar via de <a href="https://play.google.com/store/apps/details?id=be.ugent.zeus.hydra">Play Store</a> of via de <a href="https://github.com/ZeusWPI/hydra-android/releases">repository</a>.</li>
+  <li>De <strong>iOS-app</strong>, beschikbaar via de <a href="https://apps.apple.com/be/app/hydra/id602640924">App Store</a>.</li>
+  <li><strong>Hydra van de UGent</strong>, een actie (soort chat bot) voor de Google Assistent.</li>
+</ul>
+
+<hr>
+
+<h2 id="wie">Wie heeft toegang tot de verzamelde data?</h2>
+
+<p>Het Hydra-project is een project van <a href="https://www.zeus.gent">Zeus <span class="acronym">WPI</span></a> in samenwerking met de <a href="http://student.ugent.be">Dienst Studentenactiviteiten</a> van de UGent. Zeus <span class="acronym">WPI</span> beheert het project en de verzamelde data. In de eerste plaats hebben de ontwikkelaars en beheerders van het Hydra-project toegang tot de data. Daarnaast hebben de systeemadministrators van Zeus <span class="acronym">WPI</span> toegang. In uitzonderlijk geval kunnen we geanonimiseerde data delen met de UGent, voornamelijk over het gebruik van de diensten.</p>
```
Comment (redfast00) [inline server/static/website/privacy-policy.html]: www.zeus.gent -> zeus.ugent.be

File: server/static/website/privacy-policy.html [hunk 2]
```diff
@@ -1,141 +1,212 @@
...
+  <li><strong>Restolocaties.</strong> Hiervoor gebruiken we Google Maps. Indien u daar toestemming voor geeft, zal uw locatie gebruikt worden. Zelf verzamelen we deze gegevens niet. De door Google verzamelde data zijn onderhevig aan hun <a href="https://policies.google.com/privacy">privacybeleid</a>.</li>
+  <li><strong>Restomenu</strong> en <strong>artikels</strong>. Hiervoor gebruiken we onze eigen Hydra-API. Meer informatie <a href="#api">hier</a>, maar kort gezegd verzamelen we serverlogs.</li>
+  <li><strong>Informatie over bibliotheken.</strong> Raadpleeg de <a href="https://lib.ugent.be/nl/info/disclaimer">disclaimer hier</a>.</li>
+</ul>
+
+<hr>
+
+<h3 id="assistent">Assistent</h3>
+
+<p>Bij het gebruik van „Hydra van de UGent” in de Google Assistent gebruiken we deze diensten:</p>
```
Comment (redfast00) [inline server/static/website/privacy-policy.html]: Please use ascii quotes

**Comment by niknetniko** [comment]
I have applied both suggestions, my apologies for merging the PR too quickly :|

**Comment by redfast00** [comment]
No problem, I get that you want to get this done quickly :)

**Pull request merged**